src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -379,9 +377,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         argument,
                         Function(argumentValue)
                             If index >= parameters.Length - 1 AndAlso parameters.Length > 0 AndAlso parameters(parameters.Length - 1).IsParamArray Then
```

Comment:
This logic probably isn't needed. It's left over from before I discovered that the VB bound trees already include paramarray matching. I deleted the If statement and no tests failed.